src/jdk.crypto.ec/share/classes/sun/security/ec/ECPublicKeyImpl.java

```diff
@@ -176,4 +196,4 @@ long getNativePtr() {
         }
         return this.nativeECKey;
     }
-}
+}
```

Comment:
Please keep the final newline: It's present in the openj9 and openj9-staging branches and in other versions.

Reply:
Updated.